src/Microsoft.Performance.SDK.Runtime/AssemblyTypeReference.cs

```diff
@@ -63,7 +63,7 @@ protected internal AssemblyTypeReference(Type type)
 
         /// <summary>
         ///     Initializes a new instance of <see cref="AssemblyTypeReference{TDerived}"/> using
-        ///     <paramref name="other"/>'s <see cref="AssemblyTypeReference{TDerived}.Type"/>, 
+        ///     <paramref name="other"/>'s <see cref="AssemblyTypeReference{TDerived}.Type"/>,
```

Comment:
I assume your editor automatically did this, but probably best to revert this change.

Reply:
nvm, I was looking at the diff between commits. You're already touching the file.

Reply:
Changing editor settings sucks. Its not a big deal IMO.